src/editor.js

```diff
@@ -229,25 +236,40 @@ const FileListItem = ({name}, index) => html`
  * of it, to propagate a click into a hidden `input[type=file]` element.
  * @param {Event} e
  */
-function cascasdeInputClick({currentTarget}) {
+function cascadeInputClick({currentTarget}) {
   assert(currentTarget.querySelector('input')).click();
 }
 
 const dispatchUploadFiles = redispatchAs(g('upload-files'));
 
+const dispatchToggleTemplates = redispatchAs(g('toggle-templates'));
+
 /**
  * Renders a button to "upload" files--that is, set Blob URLs so they're
  * accessible from the AMP Ad document.
  */
 const FileUploadButton = () => html`
-  <div class="${n('upload-button-container')}" @click="${cascasdeInputClick}">
-    <div class="${n('upload-button')}">
+  <div class="${n('upload-button-container')}" @click="${cascadeInputClick}">
+    <div class="${n('text-button')}">
```

Comment:
Let's make `text-button` app-global (either use `@app` before `@component` on `editor.js` or add it to `global.css`).